lib/src/components/NxToggle/stateful/types.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { LabelHTMLAttributes } from 'react';
+import * as PropTypes from 'prop-types';
+
+export type Props = Omit<LabelHTMLAttributes<HTMLLabelElement>, 'onChange'> & {
+  toggleId?: string | null;
+  onChange?: ((isChecked: boolean) => void) | null;
+  defaultChecked: boolean;
+  disabled?: boolean | undefined | null;
```

Comment:
Adding `undefined` to the type union is redundant with the question mark.
```suggestion
  disabled?: boolean | null;
```